telegram/ext/filters.py

```diff
@@ -369,7 +371,59 @@ def filter(self, message):
             return bool(message.document)
 
     document = _Document()
-    """:obj:`Filter`: Messages that contain :class:`telegram.Document`."""
+    """
+    Subset for messages containing a document/file.
+
+    Examples:
+        Use these filters like: ``Filters.document.mp3``,
+        ``Filters.document.mime_type("text/plain")`` etc. Or use just
+        ``Filters.document`` for all document messages.
+
+    Attributes:
```

Comment:
This is a bit off if I render it local. This works on my side...

```suggestion
    Attributes:
        category: This Filter filters documents by their category in the mime-type attribute

            Note:
                This Filter only filters by the mime_type of the document,
                it doesn't check the validity of the document.
                The user can manipulate the mime-type of a message and
                send media with wrong types that don't fit to this handler.
        
            Examples:
                ``Filters.documents.category('audio/')`` filters all types
                of audio sent as file, for example 'audio/mpeg' or 'audio/x-wav'
                    - application: Same as ``Filters.document.category("application")``.
                    - audio: Same as ``Filters.document.category("audio")``.
                    - image: Same as ``Filters.document.category("image")``.
                    - video: Same as ``Filters.document.category("video")``.
                    - text: Same as ``Filters.document.category("text")``.
        
        mime_type: This Filter filters documents by their mime-type attribute

            Note:
                This Filter only filters by the mime_type of the document,
                it doesn't check the validity of the document.
                The user can manipulate the mime-type of a message and
                send media with wrong types that don't fit to this handler.

            Examples:
                ``Filters.documents.mime_type('audio/mpeg')`` filters all audio in mp3 format.
                    - apk: Same as ``Filters.document.mime_type("application/vnd.android.package-
                      archive")``.
                    - doc: Same as ``Filters.document.mime_type("application/msword")``.
                    - docx: Same as ``Filters.document.mime_type("application/vnd.openxmlformats-
                      officedocument.wordprocessingml.document")``.
                    - exe: Same as ``Filters.document.mime_type("application/x-ms-dos-
                      executable")``.
                    - gif: Same as ``Filters.document.mime_type("video/mp4")``.
                    - jpg: Same as ``Filters.document.mime_type("image/jpeg")``.
                    - mp3: Same as ``Filters.document.mime_type("audio/mpeg")``.
                    - pdf: Same as ``Filters.document.mime_type("application/pdf")``.
                    - py: Same as ``Filters.document.mime_type("text/x-python")``.
                    - svg: Same as ``Filters.document.mime_type("image/svg+xml")``.
                    - txt: Same as ``Filters.document.mime_type("text/plain")``.
                    - targz: Same as ``Filters.document.mime_type("application/x-compressed-
                      tar")``.
                    - wav: Same as ``Filters.document.mime_type("audio/x-wav")``.
                    - xml: Same as ``Filters.document.mime_type("application/xml")``.
                    - zip: Same as ``Filters.document.mime_type("application/zip")``.
```

Reply:
Correcting myself after discussion:
```suggestion
    Attributes:
        category: This Filter filters documents by their category in the mime-type attribute.
            
            Example:
                ``Filters.documents.category('audio/')`` filters all types
                of audio sent as file, for example 'audio/mpeg' or 'audio/x-wav'. The following
                attributes can be used as a shortcut like: ``Filters.document.audio``
                
        application:  
        audio:
        image:
        video:
        text:
        mime_type: This Filter filters documents by their mime-type attribute. 
        
            Example:
                ``Filters.documents.mime_type('audio/mpeg')`` filters all audio in mp3 format. The
                following attributes can be used as a shortcut like: ``Filters.document.jpg``
        apk: 
        doc: 
        docx:
        exe: 
        gif: 
        jpg: 
        mp3: 
        pdf: 
        py: 
        svg:
        txt:
        targz:
        wav: 
        xml: 
        zip: 
```